Tests/pageObject/RadioNavigationalWarningsAdminListEndUser.page.ts

```diff
@@ -142,6 +142,13 @@ export default class RadioNavigationalWarningsListEndUser {
     expect(beforeDetailsDateTimeGroupRnwFormat).toEqual(afterDetailsDateTimeGroupRnwFormat);
     expect(beforeDetailsRefrence).toEqual(afterDetailsRefrence);
     await this.backToAllWarning.click();
-    }
+  }
 
+  public async verifyPrint() {
+    await this.selectCheckBox.first().click();
+    await this.btnShowSelection.click();
+    await this.page.waitForLoadState();
+    expect(this.print.isEnabled()).toBeTruthy();
+    expect((await this.print.innerText()).toString()).toContain("Print");
+  }
```

Comment:
Is it possible to verify print options window has been opened after clicking print button?

Reply:
playwright interact for the web element of the page option and the print option is the window option, the playwright does interact with the window print option